### ISSUE TITLE
Radial symmetric functions

### ISSUE THREAD

**Issue by roflmaostc** [bionanoimaging/SeparableFunctions.jl#7]
I think in principle it is enough to calculate 1/8 of the array, not only one quadrant, right?

**Comment by RainerHeintzmann** [comment]
Yes, you are in principle right (in 2D) yet it is unclear whether the speed improvement with such a 1/8 scheme would outweight the memory consistency and caching advantages of the quadrant scheme, but you are more than welcome to try this out.
For the copying of the last (outermost) dimension a 1D adressing scheme is (optionally) used, as this is a bit faster than the ND copy operation.

**Comment by roflmaostc** [comment]
Depends on the function but in generell it should also perform better. (imagine a costly exp(ikr))
